internal/repository/index.go

```diff
@@ -56,8 +56,8 @@ type Index struct {
 	// only used by Store, StorePacks does not check for already saved packIDs
 	packIDToIndex map[restic.ID]int
 
-	final      bool      // set to true for all indexes read from the backend ("finalized")
-	id         restic.ID // set to the ID of the index when it's finalized
+	final      bool       // set to true for all indexes read from the backend ("finalized")
+	id         restic.IDs // set to the ID of the index when it's finalized
```

Comment:
Are the multiple id's ever actually used? Otherwise, this could be a single id and a flag indicating "merged index".

Reply:
Not yet, but will be used in #2794 :smile: